src/lib.rs

```diff
@@ -60,18 +60,23 @@ pub enum DisplayState {
     Editable,
 }
 
-/// Configure spacing and separators for the view
+///Controls the visual output of the `HexView` struct.
+///
+///There are various options which can be altered. For a detailed description of them, please see the fields below.
+///For the changes to apply, you need to use [`set_config`].
+///
+///[`set_config`]: struct.HexView.html#method.set_config
 #[derive(Debug, Clone, Copy)]
 pub struct HexViewConfig {
-    /// Number of bytes per line
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the bytes_per group value. Default is 16
     pub bytes_per_line: usize,
-    /// Number of bytes per group
+    /// Controls the number of bytes per group. It needs to be greater than 0 equal or lower than bytes_per_line. Default is 1.
     pub bytes_per_group: usize,
-    /// Separation between groups
-    pub chars_per_spacing: usize,
-    /// Separator between the address and data
+    /// Controls the separator between the hex groups in the data output. Default is ` ` (1 char long)
+    pub byte_group_separator: &'static str,
+    /// Controls the separator between the address label and the hex output of the data. Default is ` : ` (3 chars long)
```

Comment:
Please remove _"(3 chars long)"_. It serves no purpose at all

Reply:
I added it because of the spaces at either side are not very obvious when reading the doc, I thought it wouldn't hurt to clarify that the separator included those spaces. Nevermind, I have removed this in the last commit.

Reply:
Yes indeed, that's why it is in between the backticks:

    `   `

you the whitespaces will be displayed correctly and you can hover over them if you like to see how many spaces there really are.